python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -363,6 +676,21 @@ def uniform_neighbor_sample(
                     Contains the offsets of each batch in the sampling result
     """
 
+    if batch_id_list is not None or label_list is not None or label_to_output_comm_rank is not None:
```

Comment:
We probably need to raise a depracted warning here.

Reply:
it should raise a warning. That function is wrapped with the deprecated warning wrapper